src/Driver/IBMDB2/Exception/CannotCopyStreamToStream.php

```diff
@@ -9,11 +9,11 @@
 /**
  * @internal
  *
- * @psalm-immutable
+ * @phpstan-immutable
  */
 final class CannotCopyStreamToStream extends AbstractException
 {
-    /** @psalm-param array{message: string}|null $error */
+    /** @phpstan-param array{message: string}|null $error */
```

Comment:
Probably, at this point it could be just `@param`. We use this syntax in `@return` annotations in the tests data providers a lot.